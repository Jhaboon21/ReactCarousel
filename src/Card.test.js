import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";
import TEST_IMAGES from "./_testCommon.js";

it('renders without crashing', () => {
    render(
        <Card 
            caption="string describing the image"
            src="testCard1"
            currNum={0}
            totalNum={1}    
        />
    ) 
})

// need to create a snapshot
it('should match snapshot', () => {
    const {asFragment} = render(
                            <Card 
                            caption="string describing the image"
                            src="testCard1"
                            currNum={0}
                            totalNum={1}  
                            />
                        );
    expect(asFragment()).toMatchSnapshot();
})
