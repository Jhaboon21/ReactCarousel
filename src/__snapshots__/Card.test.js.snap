// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      string describing the image
    </h4>
    <img
      alt="string describing the image"
      class="Card-image"
      src="testCard1"
    />
    <small
      class="Card-small"
    >
      Image 0 of 1.
    </small>
  </div>
</DocumentFragment>
`;
